pkg/controller/servicebindingrequest/envvars/envvars_test.go

```diff
@@ -0,0 +1,123 @@
+package envvars
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestBuild(t *testing.T) {
+
+	type testCase struct {
+		name     string
+		expected map[string]string
+		src      interface{}
+		path     []string
+	}
+
+	testCases := []testCase{
+		{
+			name: "should create envvars without prefix",
+			expected: map[string]string{
+				"STATUS_LISTENERS_0_TYPE":             "secure",
```

Comment:
Should we include `STATUS_ ` here. Refer https://github.com/application-stacks/service-binding-specification/issues/26 for discussions on this.

Reply:
or we can call this out in the description.

Reply:
The `envvars` package is agnostic to the data-structure fed to it. In the case of the specification, the process *producing* the data-structure should adjust the path to produce environment variables according to the it.